SpellChecker/sources/server.js

```diff
@@ -93,9 +93,9 @@ if (cluster.isMaster) {
 	if (config.has('ssl')) {
 		const privateKey = fs.readFileSync(config.get('ssl.key')).toString();
 		const certificateKey = fs.readFileSync(config.get('ssl.cert')).toString();
-		const trustedCertificate = fs.readFileSync(config.get('ssl.ca')).toString();
+		//const trustedCertificate = fs.readFileSync(config.get('ssl.ca')).toString();
```

Comment:
Did you intend to remove this commented line?

Reply:
Left it by mistake. Thanks